financials/income_log.csv

```diff
@@ -12,3 +12,4 @@ income_id,date,type,source,currency,amount,exchange_rate,USD_equivalent,to_addre
 11,20190203,donation,Sparkpool,BTC,2.38217288,3464.01,8251.87,3B3qq3EAPtpkSoGHSDfY6rxpK5Bqh7zmij,https://www.blockchain.com/zh/btc/tx/147b921ac69042cd1bac5ca0b2715b8a2951f0bd9f27232260ed0a7a6d0563b0,Yeastplume 4th funding campaign, https://www.grin-forum.org/t/funding-campaign-yeastplume-march-to-aug-2019/1697
 12,20171214,donation,BlockCypher,BTC,0.51281968,17772.00,9113.83,3QerBBmyjjiep7o1aebBpsTDzHRMknH36m,https://www.blockchain.com/btc/tx/adef3e05799cd2af29b18c163d970caaa2c99f6381ba3bb2f921194c6aca7192, Yeastplume 1st funding campaign, https://lists.launchpad.net/mimblewimble/msg00375.html
 13,20180907,donation,BlockCypher,BTC,1.0,6445.00,6445.00,3NfmvyAMDAMCX8M5zngGqkLeKjrSg6ANz5,https://www.blockchain.com/btc/tx/65626c7ab0435b6ff202f545394b563bce9b8d83ed790bf9ee4131b2a8c63175, Yeastplume 3rd funding campaign, https://www.grin-forum.org/t/funding-campaign-yeastplume-oct-2018-to-feb-2019-closed/677
+14,20180314,donation,GGM,GRIN,630.965700337,2.70,1703.61,https://donations.grin-tech.org,a1ef9ba5-4fd8-4ffb-bcde-19c26e21b366,Dev miner fee on Grinmint grincouncil account (mostly or only GGM)

```

Comment:
I'm getting an error when trying to view this `.csv`: "It looks like row 15 should actually have 12 columns, instead of 11."